projects/dsp-ui/src/lib/viewer/values/time-value/time-input/time-input.component.html

```diff
@@ -1,11 +1,11 @@
 <div [formGroup]="form">
-    <mat-form-field class="child-input-component">
-      <dsp-jdn-datepicker [activeCalendar]="'Gregorian'">
-        <input matInput class="time-input-element date" [matDatepicker]="picker" [formControlName]="'date'" (dateChange)="_handleInput()" aria-label="Date" [placeholder]="dateLabel" readonly>
-        <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
-        <mat-datepicker #picker></mat-datepicker>
-      </dsp-jdn-datepicker>
-    </mat-form-field>
+    <dsp-jdn-datepicker [activeCalendar]="'Gregorian'">
```

Comment:
Is this change of the wrapping (datepicker directive, mat form field) intentional?

Reply:
It's intentional so that the css of the calendar icon works correctly. It fixes this https://dasch.myjetbrains.com/youtrack/issue/DSP-647. Two birds, one stone 🤷

Reply:
Ok, great!